client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -31,6 +41,58 @@ object DiagramView {
 
     private val titleVar = Var("")
 
+    def diagramControls(d: DiagramConfig): HtmlElement =
+      div(
+        cls := "flex w-full justify-around",
+        a(
+          cls := "text-4xl rounded text-center place-self-center",
+          "🔼 ",
+          onClick.map(_ => Command.MoveDiagram(d, Direction.Down)) --> Command.observer
+        ),
+        a(
+          cls := "text-4xl rounded text-center place-self-center",
+          "🔽",
+          onClick.map(_ => Command.MoveDiagram(d, Direction.Up)) --> Command.observer
+        ),
+        a(
+          cls := "text-white font-bold text-xl py-2 px-4 mx-2 rounded text-center place-self-center bg-red-500 hover:bg-red-700",
+          "Remove 💥",
+          onClick.map(_ => Command.RemoveDiagram(d)) --> Command.observer
+        )
+      )
```

Comment:
I noticed the cursor changes to an edit cursor hovering over the button. It should stay a pointer cursor though.